File: web3rpc/rpc-specs/code-samples/python/personal/sign.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

address = "0xfa415bb3e6231f488ff39eb2897db0ef3636dd32"
message = "0xdeadbeaf"
password = "helloWorld"

w3 = Web3(Web3.HTTPProvider(host))
personal_response = w3.personal.sign(message, address, password)

print(personal_response)

File: web3rpc/rpc-specs/code-samples/javascript/personal/sign.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");

  const address = "0x6b1ac7bda0073095df1d434c39d66fb31a592bdd";
  const message = "0xdeadbeaf";
  const password = "helloWorld";

  provider.personal
    .sign(message, address, password, {}, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();
